### ISSUE TITLE
Possible to check specific servers?

### ISSUE THREAD

**Issue by huantianad** [inlife/discord-global-mutual#3]
Is it possible only to check for users with mutual servers in a specific server? I'm just in too many.

**Comment by inlife** [comment]
Well, I don't see how that would work. Doesn't it kinda defeat the purpose?

**Comment by huantianad** [comment]
Let me rephrase the question: Can I specifically check for all users in two or more servers? For example, find people in both Discord Developers and Discord API without checking other servers too?

**Comment by inlife** [comment]
I would recommend adding a filter in this line, to skip all other guild ids except the ones you need:
https://github.com/inlife/discord-global-mutual/blob/master/export.js#L38

**Comment by huantianad** [comment]
OK Thanks!
